app/templating/schema_context.py

```diff
@@ -56,3 +59,7 @@ def _build_respondent(metadata):
         "trad_as": metadata["trad_as"],
         "trad_as_or_ru_name": metadata["trad_as"] or metadata["ru_name"],
     }
+
+
+def json_safe(data):
+    return data.replace('\\', r'\\').replace('"', r'\"')
```

Comment:
Just thinking it would be good to sanitise generally rather than specifically for this case to catch any other issues outside of quotes we might hit.